rpms/nodejs-electron13-bin/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) Electron contributors
```

Comment:
I think this file does not need to be copied here separately? Upstream already have this file in the source code.